cvmfs/swissknife_lease_json.h

```diff
@@ -7,6 +7,7 @@
 
 #include "swissknife_lease_curl.h"
 
+#include <cstdint>
```

Comment:
```suggestion
```
Let's keep this out of this PR - That won't work on centos7, and we may want to do one more release for that platform.